modules/virt-creating-a-vm-from-a-default-os-image.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-default-os-//images.adoc
+
+[id="virt-creating-a-vm-from-a-default-os-image_{context}"]
+= Creating a virtual machine from a default OS image
+
+After you have created a Persistent Volume Claim (PVC) for default OS images, you create a new virtual machine or virtual machine template from a default OS image that you uploaded to the PVC.
+
+[NOTE]
+====
+You can create virtual machines and virtual machine templates from default OS images with non-administrator user privileges. You need administrator privileges to upload and save default OS images to PVCs.
+====
+
+.Prerequisites
+
+* A PVC must exist that supports default OS images.
```

Comment:
this one is not needed, the second line covers this

Reply:
Deleted.